reference/common/models/alertthreshold.v1.yaml

```diff
@@ -0,0 +1,19 @@
+type: object
```

Comment:
Consider leveraging the existing definition `./data/models/blood/glucose/glucose.v1.yaml`

It splits the mg/dL and mmol/L definitions with `$oneOf` to enforce different min/max and type restrictions (integer vs. float).

Reply:
> Consider leveraging the existing definition `./data/models/blood/glucose/glucose.v1.yaml`
> 
> It splits the mg/dL and mmol/L definitions with `$oneOf` to enforce different min/max and type restrictions (integer vs. float).

Nice, thanks!

I don't see the min/max settings that you mentioned? Am I not looking in the right places? That might be for the best anyway, as the CPA has different limits than a data upload would have.

I wonder if I knew about these definitions when I was writing the code, cuz they line up too perfectly to just be coincidence... But I certainly didn't remember them, if I knew about them, when I was writing these docs... So yeah, anyway, thanks for pointing them out.

Reply:
Check this: https://github.com/tidepool-org/TidepoolApi/blob/master/reference/data/models/blood/valuemgdl.v1.yaml

Reply:
OK, but that isn't referenced from https://github.com/tidepool-org/TidepoolApi/blob/master/reference/data/models/blood/glucose/glucose.v1.yaml as far as I can tell?

I trace the above to [glucosemgdl.v1.yaml](https://github.com/tidepool-org/TidepoolApi/blob/BACK-2500-add-ability-to-send-care-partner-invites/reference/data/models/blood/glucose/glucosemgdl.v1.yaml), and from there to [../unitsmgdl.v1.yaml](https://github.com/tidepool-org/TidepoolApi/blob/BACK-2500-add-ability-to-send-care-partner-invites/reference/data/models/blood/unitsmmol.v1.yaml), where the chain ends... I'll dig around and see where the one you linked is used, and if that can be used for my purposes.

Reply:
`./glucose.v1.yaml` references `./glucosemgdl.v1.yaml` which in turn references both `../unitsmgdl.v1.yaml` (for units) and `../valuemgdl.v1.yaml` (for the value), and the min/max are in the latter. Same is true for the mmol side.

```
properties:
  units:
    $ref: '../unitsmgdl.v1.yaml'
  value:
    $ref: '../valuemgdl.v1.yaml'
```

Reply:
Oh wow, somehow I'm very blind, and didn't see the valuemgdl.v1.yaml... weird. Sorry.